src/modals/Execute.tsx

```diff
@@ -77,6 +84,19 @@ function Execute() {
         return funcData;
     };
 
+    const close = () => {
+        dispatch({ type: 'set_modals', modal: { execute: false } });
+        setResults('');
+        setExecute({
+            check: false,
+            show: false,
+            address: '',
+            value: '',
+            function: '',
+            fees: ''
```

Comment:
Could we store this into an `initialState` variable?

Reply:
updated